// ngx-leaflet/fesm2015/ngx-leaflet.mjs
import * as i0 from '@angular/core';
import { InjectionToken, NgModule } from '@angular/core';
import * as L from 'leaflet';

const NgxLeafletService = new InjectionToken('Leaflet');

class NgxLeafletModule {
}
NgxLeafletModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
NgxLeafletModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule });
NgxLeafletModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule, providers: [{ provide: NgxLeafletService, useFactory: () => L }], imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [],
                    imports: [],
                    exports: [],
                    providers: [{ provide: NgxLeafletService, useFactory: () => L }],
                }]
        }] });

/*
 * Public API Surface of ngx-leaflet
 */

/**
 * Generated bundle index. Do not edit.
 */

export { NgxLeafletModule, NgxLeafletService };
