// ngx-leaflet/esm2020/lib/ngx-leaflet.module.mjs
import { NgModule } from '@angular/core';
import { NgxLeafletService } from './ngx-leaflet.service';
import * as L from 'leaflet';
import * as i0 from "@angular/core";
export class NgxLeafletModule {
}
NgxLeafletModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
NgxLeafletModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule });
NgxLeafletModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule, providers: [{ provide: NgxLeafletService, useFactory: () => L }], imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.12", ngImport: i0, type: NgxLeafletModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [],
                    imports: [],
                    exports: [],
                    providers: [{ provide: NgxLeafletService, useFactory: () => L }],
                }]
        }] });
